.travis.yml

```diff
@@ -1,23 +1,31 @@
 language: php
 
-php:
-  - 5.5
-  - 5.6
-  - 7.0
-  - hhvm
```

Comment:
I don't like seeing this dropped... the `matrix` is supposed to be used to set additional flags for versions that are set to be tested by the `php` key.